cypress/integration/login.js

```diff
@@ -21,11 +21,50 @@ describe('Sourcebooks login', function() {
         cy.get('.page__title').contains('Timesheets');
         cy.get('.calendar').should('be.visible');
         cy.get('.tile.form').should('be.visible');
-        cy.get('.user-info__title').contains('Demo User');
+        cy.get('.user-info__title').contains('Dominykas Poškus');
         cy.get('.main-nav').find('li').should('have.length', 2);
     })
-    
+
     it('Calendar should show todays date', function(){
         cy.get('.calendar--today').find('.calendar__date').contains((new Date()).getDate());
     })
+
+    it('Verify all user roles can log in and should see appropriate tabs', function(){
+        let Roles = [
+            {
+                "name":"User",

```

Comment:
You should add a space after ":" so its easier to read actual values because now it all looks stuck together like a long spaghetti .